app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -649,11 +649,37 @@ export default Ember.Controller.extend(SelectedPostsCount, BufferedContent, {
       const quotedText = Quote.build(post, quoteState.get('buffer'));
       this.send('deselectText');
 
-      composerController.open({
-        action: Composer.CREATE_TOPIC,
-        draftKey: Composer.REPLY_AS_NEW_TOPIC_KEY,
-        categoryId: this.get('model.category.id')
-      }).then(() => {
+      var options;
+
+      if (this.get('model.isPrivateMessage')) {
+
+        var users = this.get('model.details.allowed_users');
```

Comment:
We prefer to use `let` instead of `var` since `let` will ensure that the variable is only available within the block scope.